acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/signing/BlsSigningAcceptanceTest.java

```diff
@@ -124,6 +125,18 @@ public void ableToSignUsingYubiHsm() {
     signAndVerifySignature(yubiHsmShellEnvMap());
   }
 
+  @Test
+  @Disabled("Requires access to Interlock on Armory II")
```

Comment:
can we prove this with interlock running as an app? 
edit: apperas to only have pre-builts for arm ... so maybe not

But would be better to not have this test disabled.

Reply:
Could we use wiremock or similar here so we don't have to disable the test?